### PR TITLE
Activity log alerts resource creation

### PR DESCRIPTION
**Pull request by vkagusa-mckinsey**
creating a custodian resource of Activity alert logs.

### CODE REVIEW

**Comment by thisisshi** [comment]
please add a test for this

**Comment by vkagusa-mckinsey** [comment]
> please add a test for this

test is added.

**Review by thisisshi — CHANGES_REQUESTED** [review]
looks like it's missing the actual python test file

File: tools/c7n_azure/c7n_azure/resources/alertlogs.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('alert_logs')
```
[the remark this answers is not in the document]
Comment (vkagusa-mckinsey) [reply tools/c7n_azure/c7n_azure/resources/alertlogs.py]: Done.

[the remark this answers is not in the document]
Comment (vkagusa-mckinsey) [reply tools/c7n_azure/c7n_azure/resources/alertlogs.py]: Done.

[the remark this answers is not in the document]
Comment (vkagusa-mckinsey) [reply tools/c7n_azure/c7n_azure/resources/alertlogs.py]: Done.

**Comment by dslagle** [comment]
@thisisshi I think everything has been corrected. Can you confirm?

**Review by thisisshi — APPROVED** [review]
thanks, lgtm

**Pull request merged**